codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocket08FrameEncoder.java

```diff
@@ -178,7 +179,7 @@ protected void encode(ChannelHandlerContext ctx, WebSocketFrame msg, List<Object
 
             // Write payload
             if (maskPayload) {
-                int random = (int) (Math.random() * Integer.MAX_VALUE);
+                int random = PlatformDependent.threadLocalRandom().nextInt();
```

Comment:
This was previously never negative but now it can be. That won't be a problem, right?

Reply:
From what I know it's not a problem. But that's a good point. Who knows, maybe someone already expects it to be positive :). So let's preserve the prev behavior just in case.